pkg/project/project.go

```diff
@@ -603,15 +603,23 @@ func (script *Script) SourceScript() *projectfile.Script { return script.script
 // Name returns script name
 func (script *Script) Name() string { return script.script.Name }
 
-// Language returns the language of this script
-func (script *Script) Language() language.Language {
-	return script.script.Language
+// Languages returns the languages of this script
+func (script *Script) Languages() []language.Language {
+	stringLanguages := strings.Split(script.script.Language, ",")
+	languages := make([]language.Language, len(stringLanguages))
+	for i, lang := range stringLanguages {
+		languages[i] = language.MakeByName(strings.TrimSpace(lang))
+	}
+	return languages
 }
 
-// LanguageSafe returns the language of this script. The returned
-// language is guaranteed to be of a known scripting language
+// LanguageSafe returns the first language of this script. The
+// returned language is guaranteed to be of a known scripting language
 func (script *Script) LanguageSafe() language.Language {
```

Comment:
This should return a list as well. Otherwise you're implicitly giving users of this function limited / legacy behaviour.